source/develop/release-management/features/network/networking-api-security-groups.html.md

```diff
@@ -0,0 +1,352 @@
+are used to associate a set of rules to a port.
+
+They are are indeed a white list - e.g. everything is *denied* by
+default, and the user adds rules that allow traffic. As long as *one*
+rule is matched, the traffic is allowed.
+
+Security group rules can match on L3 and L4 parameters, specifically on
+the protocol, the IP protocol, and destination ports. The remote IP
+address is also a subject for the match conditions, and rules can be
+added to allow *incoming* and *outgoing* traffic.
+
+More information on the API of security group rules can be found
+[here](https://developer.openstack.org/api-ref/network/v2/#security-group-rules-security-group-rules).
+
+### Activating the feature
+
+Two alternatives are possible:
+
+1. Add a configuration flag to the ovirt-provider-ovn configuration file.
+
+2. Use the [Allowed address pairs](https://developer.openstack.org/api-ref/network/v2/#allowed-address-pairs) networking API extension.
+Using this extension, we would be able to individually specify to which ports we would want to activate the 'deny-all' behavior, instead activating it for all ports.
+
+There's an obvious trade-off between simplicity (and developer hours)
+and granularity of the control.
+
+## Mapping networking API to OVN model objects
+
+The security group data will be model as a [port group](https://github.com/openvswitch/ovs/blob/master/ovn/ovn-nb.xml#L926).
+This ovn-nb table is only available on OVS *master* branch, and will be made available on OVS release 2.10.
+
+This new table maps a list of ACLs to a list of ports, thus eliminating
+the need to replicate the same ACL over and over, having the port in the
+match criteria as the only difference - this way, you match against a
+group of ports. More information can be found [here](http://dani.foroselectronica.es/ovn-profiling-and-optimizing-ports-creation-434/).
+
+### Encoding the security group information
+
+The security group information will be encoded in the port group object.
+
+The security group has the following attributes:
+
+| name                 | type   | description |
+|----------------------|--------|-------------|
+| tenant_id       | string | The ID of the tenant.                                                      |
+| project_id      | string | The ID of the project.                                                     |
+| description (*) | string | A human-readable description for the resource. Default is an empty string. |
+| name            | string | Human-readable name of the resource.                                       |
+
+(*) - optional attributes
+
+All of the aforementioned attributes will be stored in the external_ids
+of the corresponding port group.
+
+### Encoding the security group rule information
+
+The security group information will be encoded in the ACL object, to
+which it naturally corresponds in the OVN-world.
+
+A security group rule has the following attributes:
+
+| name                 | type   | description |
```

Comment:
From my point of view, there is no value in replicating the table from the OpenStack specification. I would be interested in the difference to the OpenStack spec, e.g. information about which attributes are implemented, and which not.